### DISCUSSION TITLE
[BUG] PDF Including Image  -- Error wrong :/ 500: The content provided is empty. Please ensure that there is text or data present before proceeding.

### DISCUSSION THREAD

**Discussion by tqangxl** [open-webui/open-webui#3266] [Ideas]
### Discussed in https://github.com/open-webui/open-webui/discussions/2713

<div type='discussions-op-text'>

<sup>Originally posted by **tqangxl** June  2, 2024</sup>
# Bug Report
@justinh-rahb  @JuanjoJG-WATA
## Description
Something went wrong :/ 500: The content provided is empty. Please ensure that there is text or data present before proceeding.
![image](https://github.com/open-webui/open-webui/assets/9669944/585bb7fc-8b82-402b-ad5f-70865b855cc5)

**Bug Summary:**
[Provide a brief but clear summary of the bug]

**Steps to Reproduce:**
[When saving a screenshot image (png, jpg, jpeg, jpe), and copying the image, a new PDF is automatically pasted from the clipboard when pasting images, saving the PDF, uploading the chat window, error.

[无标题.pdf](https://github.com/user-attachments/files/15831674/default.pdf)
[无标题jpg.pdf](https://github.com/user-attachments/files/15831675/jpg.pdf)

Operating System:
Windows 11 专业版
版本 23H2
操作系统版本 22631.3593
体验 Windows Feature Experience Pack 1000.22700.1003.0
Microsoft Edge 125.0.2535.67 (正式版本) (64 bit)
![image](https://github.com/open-webui/open-webui/assets/9669944/e517546c-defc-4289-b816-bd32b9b8755a)

[ yes] I have read and followed all the instructions provided in the README.md.
[yes ] I am on the latest version of both Open WebUI and Ollama.
[yes ] I have included the browser console logs..

Logs and Screenshots
Browser Console Logs:
"Something went wrong :/\n"There is no item named '/document.xml' in the archive""

Installation Method
stall from Open WebUI Github Repo

Requirements 📦
🐰 [Node.js](https://nodejs.org/en) = 20.13.1
🐍 [Python](https://python.org/) = 3.11.9

PS D:\Lib\Dev\AI\open-webui> python
Python 3.11.9 (tags/v3.11.9:de54cf5, Apr 2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.

PS D:\Lib\Dev\AI\open-webui> node -v
v20.13.1
PS D:\Lib\Dev\AI\open-webui>

Build and Install 🛠️
Run the following commands to install:
git clone https://github.com/open-webui/open-webui.git
cd open-webui/
cp -RPp .env.example .env
npm i
npm run build
cd ./backend
pip install -r requirements.txt -U
run open-webui1\backend\start_windows.bat
</div>

**Comment by justinh-rahb** [comment]
This is not a bug report, this is a complaint.

**Comment by justinh-rahb** [comment]
@tqangxl and stop tagging me on your complaints.
